docs/RELEASE_NOTES_CN.md

```diff
@@ -2,102 +2,81 @@
-- [改动列表](#%E6%94%B9%E5%8A%A8%E5%88%97%E8%A1%A8)
-- [v3.16](#v316)
-  - [特性](#%E7%89%B9%E6%80%A7)
-  - [特性介绍](#%E7%89%B9%E6%80%A7%E4%BB%8B%E7%BB%8D)
-    - [更好地支持 [`creator_to_cocos2dx`][1]](#%E6%9B%B4%E5%A5%BD%E5%9C%B0%E6%94%AF%E6%8C%81-creator_to_cocos2dx1)
-    - [LayerRaidalGradient](#layerraidalgradient)
-    - [Lua工程在Xcode 8.0+模拟器崩溃问题](#lua%E5%B7%A5%E7%A8%8B%E5%9C%A8xcode-80%E6%A8%A1%E6%8B%9F%E5%99%A8%E5%B4%A9%E6%BA%83%E9%97%AE%E9%A2%98)
-    - [修复iOS 11编译错误](#%E4%BF%AE%E5%A4%8Dios-11%E7%BC%96%E8%AF%91%E9%94%99%E8%AF%AF)
-    - [Cocos Studio](#cocos-studio)
-    - [提高编译速度](#%E6%8F%90%E9%AB%98%E7%BC%96%E8%AF%91%E9%80%9F%E5%BA%A6)
-    - [去除支持一些平台](#%E5%8E%BB%E9%99%A4%E6%94%AF%E6%8C%81%E4%B8%80%E4%BA%9B%E5%B9%B3%E5%8F%B0)
+- [Cocos2d-x 3.17 Release Notes](#cocos2d-x-317-release-notes)
+  - [改动列表](#%E6%94%B9%E5%8A%A8%E5%88%97%E8%A1%A8)
+  - [测试环境](#%E6%B5%8B%E8%AF%95%E7%8E%AF%E5%A2%83)
+  - [版本特性](#%E7%89%88%E6%9C%AC%E7%89%B9%E6%80%A7)
+  - [详细介绍](#%E8%AF%A6%E7%BB%86%E4%BB%8B%E7%BB%8D)
+    - [支持 iPhone X](#%E6%94%AF%E6%8C%81-iphone-x)
+    - [支持 Android Studio 3.0](#%E6%94%AF%E6%8C%81-android-studio-30)
+    - [CMake 支持全平台](#cmake-%E6%94%AF%E6%8C%81%E5%85%A8%E5%B9%B3%E5%8F%B0)
+    - [升级第三方库](#%E5%8D%87%E7%BA%A7%E7%AC%AC%E4%B8%89%E6%96%B9%E5%BA%93)
+    - [移除过时的内容](#%E7%A7%BB%E9%99%A4%E8%BF%87%E6%97%B6%E7%9A%84%E5%86%85%E5%AE%B9)
+    - [Bugs 修复，提高稳定性](#bugs-%E4%BF%AE%E5%A4%8D%E6%8F%90%E9%AB%98%E7%A8%B3%E5%AE%9A%E6%80%A7)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-# Cocos2d-x 3.16 Release Notes #
+# Cocos2d-x 3.17 Release Notes #
 
-# 改动列表
+## 改动列表
 
 * [Full Changelog](https://github.com/cocos2d/cocos2d-x/blob/v3/CHANGELOG)
 
-# v3.16
+## 测试环境
 
-该版本使用如下的Android工具链进行测试：
+Android 
 
-- Android Studio 2.3.3
-- NDK r14
+- Android Studio 3.0
+- NDK r16
 
-## 特性
+Linux
 
-- 更好地支持 [`creator_to_cocos2dx`][1] 这个Cocos Creator的插件
-- 新增 __LayerRadiaGradientLayer__
-- 支持 __Android Studio 2.3.3__
-- 修复lua工程在Xcode 8.0+模拟器崩溃问题
-- 回退CocosStudio的reader和flatbuffer
-- 修复iOS 11编译错误
-- 使用bullet的预编译库以加快编译速度
-- 去除Windows 10 metor模式、Windows Phone和Tizen的支持
-- Web引擎更新Spine runtime到v3.5.35
+- Ubuntu 16.04
 
-## 特性介绍
+Windows
 
-### 更好地支持 [`creator_to_cocos2dx`][1]
+- Visual Studio 2015
 
-[`creator_to_cocos2dx`][1]是一个[Cocos Creator][2]的插件。该插件用于导出Creator创建的场景内容给cocos2d-x的C++/Lua工程使用。下文的 
- __插件__ 就是指这个插件。
+## 版本特性
 
-通过使用该插件，可以把[Cocos Creator][2]当成cocos2d-x的场景编辑器。该插件目前支持[这些特性](3)，会持续支持其他新的特性。有些特性只在v3.16+版本支持，比如RichText img标签。
+- 支持 iPhone X
+- 支持 Android Studio 3.0
+- CMake 支持全平台，支持预编译引擎库
+- 升级 Spine runtime 至 v3.6.39
+- 升级 GLFW 至 3.2.1，并提供预编译库
+- 更新 Box2D，并提供预编译库
+- 去除 Android 的 ant 工程
+- 去除 Visual Studio 2013 的支持
 
-插件的详细使用方法可以参考插件仓库的[the READE][1]，欢迎使用并给与反馈。
+## 详细介绍
 
-![][4]
-__Scene effect in cocos creator__
+### 支持 iPhone X
 
-![][5]
-__Scene effect in cocos2d-x__
+自 iPhone X 流行，使用引擎的 iOS 开发者存在升级支持 iPhone X 的需求，该版本为开发者解决了这个问题。支持 iPhone X 全屏，启动屏幕改用 Storyboard，增加了一个获取 SafeArea 的接口，支持自动隐藏 Home 指示器。
```

Comment:
It is better to add some pictures to show the feature here. And more detail information here is needed, for example, add a link of apple docs about what's `SafeArea`, how to get it? Add some code snippet here to describe it.